src/app/integrations/landing-v2/landing-v2.component.ts

```
@@ -153,10 +146,12 @@
     }
 
     const payload = {
-      callbackUrl: this.integrationCallbackUrlMap[accountingIntegrationApp][0],
-      clientId: this.integrationCallbackUrlMap[accountingIntegrationApp][1]
+      callbackUrl: integrationCallbackUrlMap[accountingIntegrationApp][0],
+      clientId: integrationCallbackUrlMap[accountingIntegrationApp][1]
     };
 
+    console.log({payload});
```

ESLint

[error] Unexpected console statement.

(no-console)